api/src/pcapi/core/offers/api.py

```diff
@@ -100,6 +100,7 @@
 from pcapi.utils.rest import load_or_raise_error
 from pcapi.workers.push_notification_job import send_cancel_booking_notification
 
+from ..categories.subcategories import Subcategory
```

Comment:
(style / question de goût) utilise `.` pour accéder à un module du même sous-package me semble raisonnable. Accéder à un module dans un autre sous-package via `..` (ou plus !) me semble bien moins lisible. Je verrais plutôt le chemin complet, dans ce cas : 
 
```suggestion
from pcapi.core.categories.subcategories import Subcategory
```

Reply:
L'IDE a fait l'import et je n'ai pas vérifié.
Complètement d'accord avec toi.